src/scripts/main.js

```diff
@@ -353,8 +353,21 @@ const people = [
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
+const table = document.querySelector('table');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((el) => {
```

Comment:
You are directly modifying the `people` array. It's a good practice to avoid mutating data directly. Consider creating a new array with the modified data.